eng/devices/windows.cake

```diff
@@ -24,7 +24,7 @@ var DEVICETEST_APP = Argument("devicetestapp", EnvironmentVariable("WINDOWS_DEVI
 FilePath TEST_APP_PROJECT = Argument("appproject", EnvironmentVariable("WINDOWS_TEST_APP_PROJECT") ?? (!string.IsNullOrEmpty(DEFAULT_APP_PROJECT) ? DEFAULT_APP_PROJECT : PROJECT));
 FilePath DEVICETEST_APP_PROJECT = Argument("appproject", EnvironmentVariable("WINDOWS_DEVICETEST_APP_PROJECT") ?? PROJECT);
 var TEST_RESULTS = Argument("results", EnvironmentVariable("WINDOWS_TEST_RESULTS") ?? "");
-string CONFIGURATION = Argument("configuration", "Debug");
+string CONFIGURATION = Argument("configuration", "Release");
```

Comment:
should we use a helper method here GetBuildVariable ? also can we make it Uppercase like above?